### PR TITLE
CORE-5958 Disable registration status endpoint

### PR DESCRIPTION
**Pull request by YashNabar**
https://r3-cev.atlassian.net/browse/CORE-5958
Temporarily disable registration status endpoint, until it is implemented in CORE-5957.

### CODE REVIEW

File: components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/MemberRegistrationRpcOpsTest.kt [hunk 1]
```diff
@@ -78,20 +78,20 @@ class MemberRegistrationRpcOpsTest {
...
-    @Test
-    fun `checking registration progress calls the client svc`() {
-        memberRegistrationRpcOps.start()
-        memberRegistrationRpcOps.activate("")
-        memberRegistrationRpcOps.checkRegistrationProgress(HOLDING_IDENTITY_ID)
-        verify(memberOpsClient).checkRegistrationProgress(eq(HOLDING_IDENTITY_ID))
-        memberRegistrationRpcOps.deactivate("")
-        memberRegistrationRpcOps.stop()
-    }
+//    @Test
```
Comment (charlieR3) [inline components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/MemberRegistrationRpcOpsTest.kt:82]: Could you add that comment here also that links to the Jira number?

**Review by charlieR3 — DISMISSED** [review]
Looks good but I think the swaggerBaseline.json file will need to be updated to allow the tests to pass.

File: components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/MemberRegistrationRpcOpsTest.kt (hunk 1 above)
Comment (YashNabar) [reply components/membership/membership-http-rpc-impl/src/test/kotlin/net/corda/membership/impl/httprpc/v1/MemberRegistrationRpcOpsTest.kt:82]: Done

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by charlieR3 — APPROVED** [review]
LGTM

**Pull request merged**